rebalance/rebalance.py

```diff
@@ -719,7 +722,7 @@ def rebalancereport(plugin: Plugin):
     res["total_rebalanced_amount"] = total_amount
     res["total_rebalance_fee"] = total_fee
     if total_amount > Millisatoshi(0):
-        res["average_rebalance_fee_ppm"] = round(total_fee/total_amount*10**6, 2)
+        res["average_rebalance_fee_ppm"] = round(total_fee / total_amount * 10**6, 2)
```

Comment:
How do you set up/run flake8? I've checked this before committing this code, but it did not say anything to me.

Reply:
I have a file `ln_setup.sh` which I source when developing lightning an plugins.
It sets various aliases, in this case:

```
alias ln_check_python="flake8 --ignore=E501,E731,W503,W504"
```

Then I just use i.e. `ln_check_python rebalance/*.py` when I made changes in rebalance.

-------------

Also you might love that:

```
# tests a specified file and optional testcase
function ln_test() {
    if [ -z $1 ]; then
        echo "Usage: needs at least a pyton testfile as first argument or optionally a specific testcase name as second argument or both of them concatenated with '::'"
        exit 1
    fi
    if [[ "$1" =~ "::" ]]; then
        set -- $(echo "$1" | awk '{ gsub(/::+/, "\n" ); print; }' | head -n1) $(echo "$1" | awk '{ gsub(/::+/, "\n" ); print; }' | tail -n1)
    fi
    if [ -z $2 ]; then
        py.test -n 8 -d -v $1
    else
        py.test -n 8 -d -v $1 -k $2
    fi
}

# run 50 times on ${MAKE_PAR} threads
function ln_test_flaky() {
    if [ -z $1 ]; then
        echo "Usage: needs at least a pyton testfile as first argument or optionally a specific testcase name as second argument or both of them concatenated with '::'"
        exit 1
    fi
    if [[ "$1" =~ "::" ]]; then
        set -- $(echo "$1" | awk '{ gsub(/::+/, "\n" ); print; }' | head -n1) $(echo "$1" | awk '{ gsub(/::+/, "\n" ); print; }' | tail -n1)
    fi
    if [ -z $2 ]; then
        py.test --count 100 -x -n ${MAKE_PAR} -v $1
    else
        py.test --count 100 -x -n ${MAKE_PAR} -v $1 -k $2
    fi
}

# Converts Python dict outputs in a way that JSON jq likes it 
function ln_jsonify() {
     echo $1 | sed  "s/'/\"/g" | sed "s/False/false/g" | sed "s/True/true/g" | sed "s/None/null/g" | sed "s/\([^\"]\)\([0-9]\+msat\)\([^\"]\)/\1\"\2\"\3/gp" | jq
}
```

Reply:
Wow, thank you, I will try these!